### ISSUE TITLE
prose example mis-match for `@who` (of `<sp>`)

### ISSUE THREAD

**Issue by sydb** [TEIC/TEI#2407]
In att.ascribed.xml the paragraph of description in the 1st `<exemplum>` says
~~~xml
        <p>In the following example from Hamlet, speeches (<gi>sp</gi>) in the body of the play 
        are linked to <gi>castItem</gi> elements in the <gi>castList</gi> using the <att>who</att>
        attribute.</p>
~~~
But if you look at the example, the `@who` attributes actually point to the `<role>` elements, not the `<castItem>`s.

**Comment by JanelleJenstad** [comment]
Easy fix (I think). I will revise the example and chapter text after Council F2F.

**Comment by sydb** [comment]
Note — PR with a proposed fix is #2477.

**Comment by ebeshero** [comment]
Merged and closed. Small editorial changes like this can be pushed directly to dev.
